src/formio/templates/component.ejs

```diff
@@ -6,5 +6,5 @@
     {% if (ctx.visible) { %}
       {{ctx.children}}
     {% } %}
-  <div ref="messageContainer" class="{{ctx.ofPrefix}}errors"></div>
+  <div ref="messageContainer" class="{{ctx.ofPrefix}}errors" aria-live="assertive"></div>
```

Comment:
something like this might help:

```{% if (ctx.component.validateOn == 'blur') %}assertive{% } else { %}polite{% } %}``` (untested)

this could also work: `{{ ctx.component.validateOn == 'blur' ? 'assertive' : 'polite' }}` (also untested)